### ISSUE TITLE
keras.metrics.AUC() for a ternary classification problem

### ISSUE THREAD

**Issue by masih4** [keras-team/keras#13491]
I am using Keras 2.3.1 For a ternary (3 classes) classification problem. I use the `keras.metrics.AUC()` as the metric. I see val_auc_3 at the end of each epoch. I am wondering to know if "val_auc_3" is the average AUC (i.e. [auc(class1 vs all) + auc(class2 vs all) +auc(class3 vs all)]/3) or is is the AUC of the third class (i.e. just third-class vs. all)?
Is there any way to disregard one the AUCs?
